src/go/rpk/pkg/cli/profile/create.go

```diff
@@ -111,14 +121,15 @@ func createCtx(
 	fs afero.Fs,
 	y *config.RpkYaml,
 	cfg *config.Config,
-	fromSimple string,
+	fromRedpanda string,
+	fromProfile string,
 	fromCloud string,
 	set []string,
 	name string,
 	description string,
 ) (cloudMTLS, cloudSASL bool, err error) {
-	if fromCloud != "" && fromSimple != "" {
-		return false, false, fmt.Errorf("cannot use --from-cloud and --from-simple together")
+	if (fromCloud != "" && fromRedpanda != "") || (fromCloud != "" && fromProfile != "") || (fromRedpanda != "" && fromProfile != "") {
+		return false, false, fmt.Errorf("cannot only use one of --from-cloud, --from-redpanda, or --from-profile")
```

Comment:
can you provide a comment explaining the logic here?

Reply:
If I didn't make a mistake in the error message wording, it'd be self documenting
```
can only use one of ...
```
I'll fixup the error